backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/inventory/InventoryLineHU.java

```diff
@@ -213,4 +219,35 @@ public HuId getHuIdNotNull()
 
 		return huId;
 	}
+
+	public static InventoryLineHU add(@NonNull final InventoryLineHU hu1, @NonNull final InventoryLineHU hu2)
+	{
+		Check.assumeEquals(hu1.getHuIdNotNull(), hu2.getHuIdNotNull());
+		if (hu1.getId() != null && hu2.getId() != null)
+		{
+			throw new AdempiereException("Can't add InventoryLineHU that's already been persisted");
+		}
```

Comment:
this logic is above my paygrade..could you maybe add some comments or explain it in a javadoc?